test/govspeak_attachments_test.rb

```diff
@@ -344,59 +344,6 @@ def render_govspeak(govspeak, attachments = [], options = {})
     end
   end
 
-  test "a full attachment rendering looks correct" do
```

Comment:
I see what you mean - there are lots of tests in this file already that seem to be checking the same things :-).